content/en/database_monitoring/setup_sql_server/azure.md

```diff
@@ -179,7 +179,7 @@ The recommended ODBC driver is [Microsoft ODBC Driver][8]. Starting with Agent 7
 
 ```yaml
 connector: odbc
-driver: '{ODBC Driver 18 for SQL Server}'
+driver: 'ODBC Driver 18 for SQL Server'
```

Comment:
Is this a placeholder or a literal string?

Reply:
Placeholder, but you can ignore this for now. I forgot to add the "do not merge" label.